terraform/production/main.tf

```diff
@@ -203,11 +203,6 @@ moved {
   to   = module.application_roles.aws_iam_role_policy_attachment.task_execution_managed_policy
 }
 
-moved {
-  from = module.application.aws_iam_role_policy.parameter_access
-  to   = module.application_roles.aws_iam_role_policy.parameter_access
-}
-

```

Comment:
Seems to be a cyclic dependency when considered with the another moved block (on line 110) as below, which is the exact opposite 

```
moved {
  from = module.application_roles.aws_iam_role_policy.parameter_access
  to   = module.application.aws_iam_role_policy.parameter_access
}
```

The one on line 110 is correct so this can be removed

Reply:
Did this move and get moved back again or something?

Reply:
Yeah looks like it from the commit history. The moved blocks would have existed as and when needed in staging while being built progressively, but not in prod